Agoda.Frameworks.Http.AutoRestExt/ExecuteResult.cs

```diff
@@ -57,12 +57,12 @@ private Exception BuildAggregateException()
         {
             if (Exceptions.Count <= 0) return null;
 
-            var sb = new StringBuilder("There are some exceptons occurred");
+            var sb = new StringBuilder("Exceptions: ");
             foreach (var ex in Exceptions)
             {
-                sb.Append(string.Format("\r\n - \"{0}\" -> {1}", ex.Uri, ex.Message));
+                sb.Append(string.Format("\r\n - \"{0}\" -> {1}", ex.Message, ex.Uri));
```

Comment:
I am not sure why you want to change this part, as it's mostly for backward compatibility (old internal component).